### ISSUE TITLE
(Sitemap) Dependency failure if sitemap not enabled

### ISSUE THREAD

**Issue by Peekmo** [symfony-cmf/SeoBundle#216]
Hello,

If the sitemap is not enabled, there's an error because the service cmf_seo.sitemap.phpcr_provider depends on cmf_seo.sitemap.default_change_frequency which is not loaded if the sitemap is not enabled in configuration.

```
The service "cmf_seo.sitemap.phpcr_provider" has a dependency on a non-existent parameter "cmf_seo.sitemap.default_chan_frequency".
```

Regards,
Peekmo

**Comment by dbu** [comment]
ups, thanks for spotting. @ElectricMaxxx so we probably should move that service into a specific configuration file after all.

**Comment by ElectricMaxxx** [comment]
Oh. I will try to get aware of it and fetch that into a test too.

**Comment by ElectricMaxxx** [comment]
@Peekmo is that bug still there?  Should have been fixed in #219 and in the final PR #225 Please reopen, when it still exists.
